package.json

```diff
@@ -80,7 +80,9 @@
   "types": "./dist/cookie/index.d.ts",
   "files": [
     "dist/*.js",
-    "dist/*.d.ts"
+    "dist/*.d.ts",
+    "dist/cookie/*.js",
+    "dist/cookie/*.d.ts"
```

Comment:
```json
    "dist/**/*.js",
    "dist/**/*.d.ts",
    "!dist/__tests__/*"
```
This would be a bit more future-proof!

Reply:
```log
npm notice 
npm notice 📦  tough-cookie@5.0.0-rc.0
npm notice === Tarball Contents === 
npm notice 1.5kB  LICENSE                         
npm notice 33.0kB README.md                       
npm notice 76B    dist/cookie/canonicalDomain.d.ts
npm notice 1.8kB  dist/cookie/canonicalDomain.js  
npm notice 474B   dist/cookie/constants.d.ts      
npm notice 1.7kB  dist/cookie/constants.js        
npm notice 4.0kB  dist/cookie/cookie.d.ts         
npm notice 23.0kB dist/cookie/cookie.js           
npm notice 150B   dist/cookie/cookieCompare.d.ts  
npm notice 2.6kB  dist/cookie/cookieCompare.js    
npm notice 4.7kB  dist/cookie/cookieJar.d.ts      
npm notice 33.9kB dist/cookie/cookieJar.js        
npm notice 67B    dist/cookie/defaultPath.d.ts    
npm notice 1.1kB  dist/cookie/defaultPath.js      
npm notice 122B   dist/cookie/domainMatch.d.ts    
npm notice 3.2kB  dist/cookie/domainMatch.js      
npm notice 111B   dist/cookie/formatDate.d.ts     
npm notice 1.5kB  dist/cookie/formatDate.js       
npm notice 861B   dist/cookie/index.d.ts          
npm notice 3.4kB  dist/cookie/index.js            
npm notice 85B    dist/cookie/parseDate.d.ts      
npm notice 7.5kB  dist/cookie/parseDate.js        
npm notice 207B   dist/cookie/permutePath.d.ts    
npm notice 1.8kB  dist/cookie/permutePath.js      
npm notice 240B   dist/getPublicSuffix.d.ts       
npm notice 3.7kB  dist/getPublicSuffix.js         
npm notice 1.6kB  dist/memstore.d.ts              
npm notice 7.7kB  dist/memstore.js                
npm notice 1.7kB  dist/pathMatch.d.ts             
npm notice 2.6kB  dist/pathMatch.js               
npm notice 105B   dist/permuteDomain.d.ts         
npm notice 2.7kB  dist/permuteDomain.js           
npm notice 3.1kB  dist/store.d.ts                 
npm notice 3.0kB  dist/store.js                   
npm notice 1.1kB  dist/utils.d.ts                 
npm notice 2.7kB  dist/utils.js                   
npm notice 1.1kB  dist/validators.d.ts            
npm notice 3.8kB  dist/validators.js              
npm notice 45B    dist/version.d.ts               
npm notice 163B   dist/version.js                 
npm notice 3.2kB  package.json                    
npm notice === Tarball Details === 
npm notice name:          tough-cookie                            
npm notice version:       5.0.0-rc.0                              
npm notice filename:      tough-cookie-5.0.0-rc.0.tgz             
npm notice package size:  39.2 kB                                 
npm notice unpacked size: 165.7 kB                                
npm notice shasum:        c189345e3391edc67e361b27fbd1f43ca6755de9
npm notice integrity:     sha512-WhFGIC/JKan3D[...]yMFIFA9w03/SQ==
npm notice total files:   41                                      
npm notice 
```

Reply:
@wjhsf good call. Changed it to:
```
"dist/**/*.js",
"dist/**/*.d.ts",
"!__tests__"
```

So that we include all the types and code in the `dist` folder & exclude all `__tests__` folders found at any level in the directory structure.